cli/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 13.6.3
+
+_Released 01/02/2024 (PENDING)_
+
+**Bugfixes:**
+
+- Updated display of assertions to escape markdown formatting and properly render bold styles for assertion values. Fixes [#24960](https://github.com/cypress-io/cypress/issues/24960) and [#5679](https://github.com/cypress-io/cypress/issues/5679).
```

Comment:
It seems this change likely fixes this issue as well: [ttps://github.com/cypress-io/cypress/issues/28100](https://github.com/cypress-io/cypress/issues/28100)

there is this [open PR](https://github.com/cypress-io/cypress/pull/28245) I pulled down to run through Percy last week but hit a broken test in the reporter so wasn't able to get percy to finalize for results.

Reply:
I don't think this would fix this issue for the actual visit command. The logic to not render in markdown is only applying for commands of name 'assert', so it would fix the format of urls that are included in assertions. It is in the same logic however, and I would think we'd want to exclude 'visit' and 'request' commands from this formatting.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/28583/commits/4e99b0c6c9c37f274390789c9229bb8665c2b496